components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImpl.java

```diff
@@ -115,7 +115,7 @@ public class OAuthAdminServiceImpl {
     public static final String AUTHORIZATION_CODE = "authorization_code";
     static final String RESPONSE_TYPE_TOKEN = "token";
     static final String RESPONSE_TYPE_ID_TOKEN = "id_token";
-    static final String BINDING_TYPE_NONE = "None";
+    static final String BINDING_TYPE_NONE = "NONE";
```

Comment:
In other places also this seems to be used as "None" any reason to use all caps?

https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java#L606